pages/en/lb4/Context.md

```diff
@@ -48,6 +51,42 @@ In this case, you are using the `.controller` helper method to register a new
 controller. The important point to note is `MyController` is actually registered
 into the Application Context (`app` is a Context).
 
+## Server-level context (components that make use of Server)
+- has the Application-level context as a parent
+- holds configuration specific to this particular server instance
```

Comment:
Server-level context:
- Is a child of application-level context.
- Hold configuration specific to a particular server instance.

---
Also, what does the parent/child relationship mean here?  I think some more explanation is in order.